examples/grafana/detail.json

```diff
@@ -598,7 +598,7 @@
               }
             ]
           },
-          "unit": "percentunit"
```

Comment:
To keep the RED (Requests, Error Rate, Duration) type of panels, how about actually using the error rate percentage?